metadata_backend/api/operators.py

```diff
@@ -597,7 +599,7 @@ def __init__(self, db_client: AsyncIOMotorClient) -> None:
         """
         super().__init__(mongo_database, "text/xml", db_client)
 
-    async def _format_data_to_create_and_add_to_db(self, schema_type: str, data: str) -> Dict:
+    async def _format_data_to_create_and_add_to_db(self, schema_type: str, data: str) -> List:
```

Comment:
When typing a list, it's nice to say what are the contents: `List[dict]` or is it `List[tuple]`?

It would also be nice if the doc string is matching the return type.